src/test/java/com/spotify/docker/client/messages/HostConfigTest.java

```diff
@@ -65,6 +66,14 @@ public void testJsonOnFailure() throws Exception {
     assertThat(hostConfig.restartPolicy(), is(HostConfig.RestartPolicy.onFailure(5)));
   }
 
+  @Test
+  public void testCapAddAndDrop() throws Exception {
```

Comment:
A bit of a dumb test, but we can't easily mock WebTarget in DefaultDockerClient right now...